browser/ui/webui/brave_education/brave_education_page_handler_unitttest.cc

```diff
@@ -0,0 +1,171 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
```

Comment:
this file should be named unittest not unitttest

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/6313ac781a12164601ca4af8668f08ca31a4860e